Makefile

```diff
@@ -1,17 +1,17 @@
-TRIVY_VERSION ?= 0.45.1
+TRIVY_VERSION ?= 0.53.0
```

Comment:
Can we remove this default value altogether and instead check that the variable is set?

None of the two GHA workflows in this repo use the default; looks like ATM this default serves no purpose and just confuses everyone when it becomes stale